pac4j-core/src/main/java/org/pac4j/core/context/J2ERequestContext.java

```diff
@@ -160,4 +160,11 @@ public String getFullRequestURL() {
         }
         return pac4jCookies;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public String getPath() {
```

Comment:
Reading the Javadoc documentation, it even seems to be unnecessary:

--

**Automatic Copying of Method Comments**

The Javadoc tool has the ability to copy or "inherit" method comments in classes and interfaces under the following two circumstances.

Inherit from classes and interfaces - Inheriting of comments occurs in all three possible cases of inheritance from classes and interfaces:

- When a method in a class overrides a method in a superclass
- When a method in an interface overrides a method in a superinterface
- When a method in a class implements a method in an interface

--

So I did the test and the comment is actually inherited without using the `{@inheritDoc}` or the `@Override` annotation.